package.json

```diff
@@ -12,6 +12,7 @@
     "autocannon": "^2.4.1",
     "coveralls": "^3.0.0",
     "http-ndjson": "^3.1.0",
+    "pino-pretty": "^3.6.1",
```

Comment:
Is this needed?

Reply:
When running `node example.js | pino` as per the README, I received the following:

`'pino' cli has been removed. Use 'pino-pretty' cli instead.`

I guess it's not _technically_ a devDependency, as they would need a global install to run the README example without npx prefix. For consistency I'll revert this.